tests/API/XCCDF/unittests/test_report_without_xsl_fails_gracefully.sh

```diff
@@ -12,6 +12,7 @@ stdout=$(mktemp -t ${name}.stdout.XXXXXX)
 # We cannot simply unset, because there still might be a non empty system wide location.
 # We can set some nonsence instead.
 OSCAP_XSLT_PATH=/blah
+export OSCAP_XSLT_PATH
```

Comment:
you can do this in one line and it probalby should go into maint-1.2 because it's not related to cmake